src/main/kotlin/OutputView.kt

```diff
@@ -0,0 +1,26 @@
+import domain.Car
+import domain.CarMoveSnapshot
+import java.lang.StringBuilder
+
+fun printSnapShots(snapshots: Map<Int, List<CarMoveSnapshot>>) {
+    println("===============라운드별 경기 결과=================")
+    for (snapshot in snapshots) {
+        println("============ ${snapshot.key} 경기 =================")
+        for (carSnapshot in snapshot.value) {
+            println("${carSnapshot.name} : ${carSnapshot.position.spread("-")}")
+        }
+    }
+    println("================================================")
+}
+
+fun printWinners(winners: List<Car>) {
+    println("축하합니다. 최종 우승자는 ${winners.joinToString(", ") { it.name }} 입니다!")
+}
+
+private fun Int.spread(type: String): String {
+    val builder = StringBuilder()
+    repeat((0 until (this)).count()) {
```

Comment:
```suggestion
    repeat(this) {
```
`until` 이나 `..`(`rangeTo`의 operator입니다) 사용도 좋지만 간단히 나타낼 수 있는건 간단히 하면 좋겠네요

Reply:
repeat 내부적으로 0 until params 로 되어 있네요 꿀팁 감사용